Modify S7000&S2091: Fix resource links (#2976)
SonarSource/rspec: rules/S6700/secrets/rule.adoc

=== Documentation

> * RapidAPI Documentation - https://web.archive.org/web/20230627151620/https://docs.rapidapi.com/docs/keys#creating-or-rotating-a-rapid-api-key[Creating or rotating a Rapid API key]

Reviewer: This link seems to work without wayback machine - did you change it on purpose anyway?

Author: Yeah, my reasoning was that anyway the wayback machine will be more reliable with the data

Author: No, actually it just didn't work at home... But it might be a Zscaler issue or maybe an issue on their side. I think I would change everything to the wayback machine but we also need to keep that updated so it's better to add the new link instead 👍

> * RapidAPI Documentation - https://web.archive.org/web/20230627151620/https://docs.rapidapi.com/docs/keys#creating-or-rotating-a-rapid-api-key[Creating or rotating a Rapid API key]

Author: ```suggestion
* RapidAPI Documentation - https://docs.rapidapi.com/docs/keys-and-key-rotation#creating-or-rotating-a-rapid-api-key[Creating or rotating a Rapid API key]
```

> * RapidAPI Documentation - https://web.archive.org/web/20230521051107/https://docs.rapidapi.com/docs/audit-trails[Audit Trails]

Reviewer: This page seems to have moved here: https://docs.rapidapi.com/docs/org-audit-trails-teams

> * RapidAPI Documentation - https://web.archive.org/web/20230521051107/https://docs.rapidapi.com/docs/audit-trails[Audit Trails]

Author: ```suggestion
* RapidAPI Documentation - https://docs.rapidapi.com/docs/org-audit-trails-teams[Audit Trails]
```

> * RapidAPI Documentation - https://web.archive.org/web/20230521051107/https://docs.rapidapi.com/docs/audit-trails [Audit Trails]

Author: ```suggestion
* RapidAPI Documentation - https://web.archive.org/web/20230521051107/https://docs.rapidapi.com/docs/audit-trails[Audit Trails]
```
